src/modules/keyboard/KeyboardLayoutModel.h

```diff
@@ -154,6 +154,20 @@ class KeyboardVariantsModel : public XKBListModel
     void setVariants( QMap< QString, QString > variants );
 };
 
+/** @brief A list of variants (xkb id and human-readable)
```

Comment:
Group switching key-combinations, not variants. The comment does not match the code.